src/nextjournal/clerk/webserver.clj

```diff
@@ -103,17 +103,15 @@
    :on-receive (fn [sender-ch edn-string]
                  (binding [*ns* (or (:ns @!doc)
                                     (create-ns 'user))]
-                   (let [{:as msg :keys [type]} (read-msg edn-string)]
+                   (let [{:as msg :keys [type sender-id]} (read-msg edn-string)]
                      (case type
                        :eval (do (eval (:form msg))
                                  (eval '(nextjournal.clerk/recompute!)))
                        :swap! (when-let [var (resolve (:var-name msg))]
                                 (apply swap! @var (eval (:args msg)))
-                                (doseq [ch @!clients :when (not= sender-ch ch)]
-                                  (httpkit/send! ch edn-string))
```

Comment:
I believe this to be redundant now that we always reset synced atoms within `n.clerk.render/intern-atoms!`. This happens by broadcasting the whole document to all clients via the recompute below.